src/Action/action_lib/action_u1_dm.jl

```diff
@@ -56,8 +56,20 @@ function init_action!(
     return nothing
 end
 
+# set repulsion for su2 symmetry
+function set_repulsion!(
+    A :: Float64,
+    a :: Action_u1_dm
+    ) ::Nothing
+
+    # init bare action on-site with level repulsion for Γxx
+    a.Γ[1].bare[1] = A / 4.0 / (2.0 * a.S)
 
+    # init bare action on-site with level repulsion for Γzz
+    a.Γ[1].bare[1] = A / 4.0 / (2.0 * a.S)
```

Comment:
Typo:

```suggestion
    # init bare action on-site with level repulsion for Γzz
    a.Γ[2].bare[1] = A / 4.0 / (2.0 * a.S)
```

Reply:
Obviously that is wrong, good spot.

Reply:
Does this work? The u1-dm type has no field S as this construction only works for Heisenberg models.